diff --git a/config.def.h b/config.def.h
index 0d6f4a4..8b84963 100644
--- a/config.def.h
+++ b/config.def.h
@@ -1,13 +1,13 @@
 /* See LICENSE file for copyright and license details. */
 
 /* appearance */
-static char *font         = "monospace:size=9";
-static char *normbgcolor  = "#222222";
-static char *normfgcolor  = "#cccccc";
-static char *selbgcolor   = "#555555";
-static char *selfgcolor   = "#ffffff";
-static char *urgbgcolor   = "#111111";
-static char *urgfgcolor   = "#cc0000";
+static char *font         = "FantasqueSansM Nerd Font:size=12";
+static char *normbgcolor  = "#2d2d2d";
+static char *normfgcolor  = "#d3d0c8";
+static char *selbgcolor   = "#d3d0c8";
+static char *selfgcolor   = "#2d2d2d";
+static char *urgbgcolor   = "#f2777a";
+static char *urgfgcolor   = "#2d2d2d";
 static char before[]      = "<";
 static char after[]       = ">";
 static char titletrim[]   = "...";
@@ -148,4 +148,4 @@ static const Key keyreleases[] = {
 	#endif // HIDETABS_PATCH
 
 };
-#endif // KEYRELEASE_PATCH
\ No newline at end of file
+#endif // KEYRELEASE_PATCH
